tests/erp/export/test_export_unit.py

```diff
@@ -0,0 +1,73 @@
+import csv
+import os
+from io import StringIO
+from typing import List
+
+import pytest
+
+from erp.export.export import export_schema_to_csv
+from erp.export.generate_schema import generate_schema
+from erp.export.utils import map_erps_to_json_schema
+from erp.export.mappers import EtalabMapper
+from erp.models import Erp
+from tests.erp.test_managers import create_test_erp
+
+
+@pytest.fixture
+def example_data(db) -> List[Erp]:
+    return [
+        create_test_erp(
+            "test 1", transport_station_presence=True, commentaire="simple commentaire"
+        ),
+        create_test_erp(
+            "test 2",
+            transport_station_presence=False,
+            commentaire="simple commentaire 2",
+        ),
+    ]
+
+
+def test_export_to_csv(example_data):
+    first_row = EtalabMapper.headers()
+    headers, mapped_data = map_erps_to_json_schema(example_data, EtalabMapper)
+    file = StringIO()
+
+    export_schema_to_csv(file, example_data, EtalabMapper)
+    file.seek(0)
+    reader = csv.DictReader(file, fieldnames=first_row)
+    next(reader)  # Skip headers
+
+    erp_0 = next(reader)
+    assert erp_0["transport_station_presence"] == str(
+        mapped_data[0].transport_station_presence
+    )
+    # assert erp_0["commentaire"] == str(mapped_data[0].commentaire)
+    erp_1 = next(reader)
+    assert erp_1["transport_station_presence"] == str(
+        mapped_data[1].transport_station_presence
+    )
+    # assert erp_1["commentaire"] == str(mapped_data[1].commentaire)
+
+    # Validate data with schema
```

Comment:
Can this patch land without this being implemented? What's missing? If we don't plan on landing this patch with this part, could we remove the comment? Or reformat the comment expliciting that this needs implementation, and why?